ios/sdk/src/callkit/JMCallKitProxy.swift

```diff
@@ -62,6 +62,8 @@ import Foundation
     @objc public static func configureProvider(localizedName: String,
                                                ringtoneSound: String?,
                                                iconTemplateImageData: Data?) {
+        guard enabled else { return }
```

Comment:
Can you explain why those flags are needed in this file at all ? It feels like those calls should not happen in the first place if CallKit is disabled. I mean they should not come from the JS side. I guess I'm missing something here ?